gameboard/setupships.js

```diff
@@ -0,0 +1,90 @@
+let ships = [
```

Comment:
Same as before. WDYT about creating `src/mocks` or if it is a `const` needed in the end script ` src/constants` and create there a file `game.js` and hold there `const` used for functions? It will be a nice and clear way to show what it is for and what it is.